src/ansiblelint/skip_utils.py

```diff
@@ -193,3 +197,12 @@ def normalize_tag(tag: str) -> str:
         used_old_tags[tag] = RENAMED_TAGS[tag]
         return RENAMED_TAGS[tag]
     return tag
+
+
+def is_nested_task(task: Dict[str, Any]) -> bool:
+    """Check if task includes block/always/rescue."""
+    for key in NESTED_TASK_KEYS:
+        if task.get(key):
+            return True
+
+    return False
```

Comment:
Thank you. Yes, it is a duplicate now.

If `from ansiblelint.utils import is_nested_task` is added, I faced the following error on `test_import[ansiblelint.__main__]`:
```
...
>       assert result.returncode == 0
E       assert 1 == 0
E        +  where 1 = CompletedProcess(args=['/usr/local/Caskroom/miniconda/base/envs/env39/bin/python', '-c', "import ansiblelint.__main__, sys; print('ansible' in sys.modules); sys.exit(0 if 'ansible' not in sys.modules else 1)"], returncode=1).returncode
```

I'm not sure but it can be caused because `utils.py` imports some modules related to ansible.
Anyway I think we need to move the `is_nested_task` from `ansiblelint.utils` to `ansiblelint.skip_utils` and add `from ansiblelint.skip_utils import is_nested_task` in `utils.py`.
I received the [this comment](https://github.com/ansible/ansible-lint/pull/2113#issuecomment-1122042380) so I leave both is_nested_task but I'd like to update as above later.
Does that make sense?

Reply:
added some tests in commit [2c26c22](https://github.com/ansible/ansible-lint/pull/2113/commits/2c26c22d3ebdc070af8cfb6b17b2fa4c7d1a5b63) but `test_is_nested_task` is a duplicate as well now.